src/main/java/uk/gov/digital/ho/pttg/application/HmrcClient.java

```diff
@@ -26,8 +31,12 @@
     private static final String PAYE_EMPLOYMENT = "paye";
     private static final String SA_SELF_EMPLOYMENTS = "selfEmployments";
 
-    public HmrcClient(HmrcHateoasClient hateoasClient) {
+    public HmrcClient(HmrcHateoasClient hateoasClient,
+                      Clock clock,
+                      @Value("#{${hmrc.self-assessment.tax-years.history.maximum:1000}}") int maximumSelfAssessmentTaxYearHistory) {
```

Comment:
Why hash and dollar?  I thought # was required only if you are doing some kind of complex expression and $ is ok for plain old property values?

Reply:
An understanding issue on my part. I've fixed it.